lib/python/qmk/commands.py

```diff
@@ -61,3 +64,20 @@ def parse_configurator_json(configurator_file):
     user_keymap = json.load(configurator_file)
 
     return user_keymap
+
+
+def run(command, *args, **kwargs):
+    """Run a command with subprocess.run
+    """
+    if isinstance(command, str):
+        raise TypeError('`command` must be a sequence such as list or tuple.')
+
+    platform_id = platform.platform().lower()
+
+    if 'windows' in platform_id:
+        command = map(shlex.quote, command)
+        # On Windows, we want to run stuff (especially scripts)
+        # in a Posix shell
+        command = ['C:/msys64/usr/bin/sh', '-c', command]
```

Comment:
Some users may not be using the x86_64 MSYS, or have installed it to some other directory.
I think `$SHELL` is actually more reliable here.

Reply:
I thought of using `os.environ['MSYSTEM_PREFIX'] + '/bin/sh'`, but it resolves to `C:/msys64/mingw64/bin/sh` which does not exist. I don't see an environment variable that just provides the MSYS base directory, so `os.environ['SHELL']` is the only way.